test/test_quantization.py

```diff
@@ -14,7 +14,7 @@
 from common_utils import run_tests
 from common_quantization import QuantizationTestCase, SingleLayerLinearModel, \
     SkipQuantModel, QuantStubModel, \
-    ModelForFusion, ManualLinearQATModel, ManualConvLinearQATModel, \
+    ModelForFusion, ModelForFusion2, ManualLinearQATModel, ManualConvLinearQATModel, \
```

Comment:
nit: can you rename ModelForFusion and ModelForFusion2 to more informative names